# Call
string task
string req
---
# Return
zeabus_vision/vision_flare data